<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>589</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>121</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>121</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>121</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>121</red>
        <green>121</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>900</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MoveImageY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>800</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="MoveImageX">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ScaleImageX">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="ScaleImageY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>800</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ExportFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Export LPC File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Filename_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set X Position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Filename_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Y Position:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Position:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Filename_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set X Scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Filename_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Y Scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Filename_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Representation of LPC Grid: (1200mm, 800mm)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Close">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Creator</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>buttonLoadImage</zorder>
   <zorder>MoveImageY</zorder>
   <zorder>MoveImageX</zorder>
   <zorder>ScaleImageX</zorder>
   <zorder>ScaleImageY</zorder>
   <zorder>ExportFile</zorder>
   <zorder>lbl_Filename_2</zorder>
   <zorder>lbl_Filename_3</zorder>
   <zorder>lbl_Filename_4</zorder>
   <zorder>lbl_Filename_5</zorder>
   <zorder>lbl_Filename_6</zorder>
   <zorder>btn_Close</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLPC_Instruction_Creator">
    <property name="title">
     <string>LPC Instruction Creator</string>
    </property>
   </widget>
   <addaction name="menuLPC_Instruction_Creator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
